File: node/static_code/server/service/helloService.js
const { messages } = require('proto')

const sayHello = (call, callback) => {
    let reply = new messages.HelloReply();
    const name = call.request.getName();
    reply.setMessage(`hello ${name}, this is node static code gen server`)
    callback(null, reply)
}

module.exports = {
    sayHello
}

File: java/grpc-demo-server/src/main/java/com/grpc/demo/lqm/service/HelloService.java
package com.grpc.demo.lqm.service;

import io.grpc.stub.StreamObserver;
import lqm.demo.grpc.server.hello.HelloServiceGrpc;
import lqm.demo.grpc.server.hello.HelloServiceOuterClass;

public class HelloService extends HelloServiceGrpc.HelloServiceImplBase {
    @Override
    public void sayHello(HelloServiceOuterClass.HelloRequest request,
                         StreamObserver<HelloServiceOuterClass.HelloReply> responseObserver) {
        final String name = request.getName();
        System.out.printf("request name is %s%n", name);
        final String helloMsg = String.format("Hello %s, this is java server", name);
        final HelloServiceOuterClass.HelloReply reply =
                HelloServiceOuterClass.HelloReply.newBuilder().setMessage(helloMsg).build();
        responseObserver.onNext(reply);
        responseObserver.onCompleted();
    }
}
